README.md

```diff
@@ -103,67 +103,8 @@ Pull request and release process:
 # Commands
```

Comment:
In order for this file to be generated correctly - please follow the guide in README for Contributing

Reply:
This file is generated after commit. Maybe I should review this process so this kind of PR's would not require building the project